perfdash/google-gcs-downloader.go

```diff
@@ -22,65 +22,49 @@ import (
 	"os"
 
 	"k8s.io/contrib/test-utils/utils"
-	"k8s.io/kubernetes/pkg/util/sets"
 )
 
 // constants to use for downloading data.
 const (
-	jobName = "kubernetes-e2e-gce-scalability"
 	logFile = "build-log.txt"
 )
 
 // GoogleGCSDownloader that gets data about Google results from the GCS repository
 type GoogleGCSDownloader struct {
-	startFrom            int
+	Builds               int
```

Comment:
The reason is that different Jenkins projects has different build number. Now we may download data from different Jenkins projects, we have to:
1) Specify startFrom for each Jenkins projects.
2) Just say we need the latest X builds, please download them from each Jenkins projects.

I think the 2) one is more straight forward, right? ;)